packages/web-react/src/types/textFieldBase.ts

```diff
@@ -1,23 +1,26 @@
-import { ChildrenProps, TextInputProps } from './shared';
+import { ChildrenProps, HelperTextProps, PasswordToggleAdornmentProp, TextInputProps, Validation } from './shared';
+import { LabelProps } from './label';
+import { MessageProps } from './message';
 import { TextAreaProps } from './textArea';
-import { TextFieldProps, TextFieldType } from './textField';
-import { PasswordToggleAdornmentProp } from './shared/adornments';
+import { TextFieldProps } from './textField';
 
 export interface TextFieldBaseMultiLineProps {
   /** Whether the input is TextArea. */
   isMultiline?: boolean;
 }
 
-export interface TextFieldBaseProps extends ChildrenProps, TextFieldProps, TextAreaProps, TextFieldBaseMultiLineProps {}
+export type TextFieldBaseCommonProps = ChildrenProps &
```

Comment:
Why not an interface?

Reply:
Because of empty object when using interface.
`export interface TextFieldBaseCommonProps
  extends ChildrenProps,
    LabelProps,
    MessageProps,
    HelperTextProps,
    TextInputProps,
    Validation {}`